internal/cli/logs.go

```diff
@@ -99,7 +101,14 @@ func (c *LogsCommand) Run(args []string) int {
 }
 
 func (c *LogsCommand) Flags() *flag.Sets {
-	return c.flagSet(flagSetOperation, nil)
+	return c.flagSet(flagSetOperation, func(set *flag.Sets) {
+		f := set.NewSet("Command Options")
+		f.StringVar(&flag.StringVar{
+			Name:   "deployment",
```

Comment:
Is this expected to be the sequence number? Maybe the flag could be `-deployment-seq`. I could see people trying deployment UUID